initiate_standard_product_acqlist_evaluator.py

```diff
@@ -229,7 +229,7 @@ def main():
             acq_info[acq] = get_acq_object(acq, "master")
         for acq in acqlist['metadata']['slave_acquisitions']:
             acq_info[acq] = get_acq_object(acq, "slave")
-        if all_slcs_exist(acq_info.keys(), acq_version, slc_version):
+        if all_slcs_exist(list(acq_info.keys()), acq_version, slc_version):
```

Comment:
This change was made as python3.7 returns dict.keys() as dict_key object. Needs to cast to list.
Trace back before change here: https://gist.github.com/shitong01/50ad08cdd91f590759e8efaf166b339a#file-evaluator-json-serializable-issue-in-standard_product-L338-L366